tests/app/data_model/test_app_models.py

```diff
@@ -16,14 +15,17 @@ def test_submitted_response(self):
         self._test_model(SubmittedResponse('txid', 'somedata', NOW))
 
     def test_questionnaire_state(self):
-        new = self._test_model(QuestionnaireState('someuser', 'somedata', 1))
-        self.assertGreaterEqual(new.created_at, NOW)
-        self.assertGreaterEqual(new.updated_at, NOW)
+        new_model = self._test_model(QuestionnaireState('someuser', 'somedata', 1))
+
+        self.assertGreaterEqual(new_model.created_at, NOW)
```

Comment:
We do have `freezegun` in the dependencies now, which you could use here. Not particularly important in this case, but worth bearing in mind if you're writing these kinds of tests.

Reply:
Will have a look into it.

Reply:
We use it in `test_date_reference_ranges.py`

Reply:
https://github.com/ONSdigital/eq-survey-runner/blob/master/tests/integration/mci/test_date_reference_ranges.py#L4